Mission-Vision-Statement.md

```diff
@@ -0,0 +1,10 @@
+# Mission and Vision
+The mission of this project is to help increase diversity and inclusion (D&I) in the workplace using the open-source approach. We hope to deliver a framework that can be used by all organizations that want to implement, improve or validate their D&I efforts. We will leverage the entire community's knowledge and experience to build and test the framework that we propose. The open-source approach we will provide everyone better visibility of D&I efforts within organizations using our framework
+
+Our vision is to provide organizations a gold standard on how to successfully implement and maintain a diverse culture and workplace. We hope to fill a knowledge gap in small organizations that do not have the means to stand-up their own D&I infrastructure. For medium and large organizations, they can use our framework to validate and possibly improve their D&I implementations. Our team will engage with these organizations, gather results from their efforts, and then integrate those results into the next iteration. We will maintain a continuous feedback loop that incorporates those deliverables as well as inputs from non-organizational engagement. 
```

Comment:
Our vision is to provide organizations a gold standard on how to successfully implement and maintain a diverse culture and workplace. We hope to fill a knowledge gap in small organizations that do not have the means to stand-up their own D&I infrastructure. For medium and large organizations, they can use our framework to validate and possibly improve their D&I implementations. Our team will engage with these organizations, gather results from their efforts, and then integrate those results into the next iteration. We will maintain a continuous feedback loop that incorporates those deliverables as well as inputs from non-organizational engagement.

Reply:
Updating statements to include feedback